### DISCUSSION TITLE
Livewire Screen Casts

### DISCUSSION THREAD

**Discussion by wcoy7** [livewire/livewire#7503] [1. Help] [answered]
So if you purchased the V3 screencasts you can't view the V2 screencasts?

**Comment by joshhanley** [comment]
@wcoy7 are you having issue accessing the V2 screencasts now or are you just asking a question?

**Reply by wcoy7** [reply]
Yes,I can access the V3 screencasts but cannot access the V2 screencasts.  Are they not included?   Not sure how else to ask this question.  V2 states you have to sponsor the project on GitHub.

**Reply by joshhanley** [reply]
@wcoy7 that was my reply from two days ago. Please see my reply below. https://github.com/livewire/livewire/discussions/7503#discussioncomment-7825057

**Comment by wcoy7** [comment]
I got the V3 screencasts, but cannot view the V2 screencasts.   So that’s my question.   You have to sponsor on GitHub still to get access to them?

— Warren

**Reply by joshhanley** [reply]
@wcoy7 ah yep, I will find out for you.

**Comment by joshhanley** [comment, marked as answer]
@wcoy7 you should now have access to v2 screencasts. Can you test and let us know if it works? Thanks!

**Reply by joshhanley** [reply]
@wcoy7 we have given your account access to V2 screencasts, so can you please test it again and let us know if it works? Thanks!

**Comment by wcoy7** [comment]
It allows for most but the filepond is still showing the you have to sponsor on Github

**Comment by wcoy7** [comment]
I just checked and any of the ones that say you have to sponsor on GitHub do not play.  So the answer is if you purchase Livewire 3 you don't have access to Livewire 2 content?

**Reply by joshhanley** [reply]
@wcoy7 not by default, no. But Caleb said he gave your username access to the V2 screencasts since you asked. Will get him to check again.
